formats/shared/default_properties/notification.jsonnet

```diff
@@ -14,6 +14,12 @@
   payload_version: {
     "$ref": "#/definitions/payload_version",
   },
+  access_limited: {
+    "$ref": "#/definitions/access_limited",
+  },
+  auth_bypass_ids: {
+    "$ref": "#/definitions/guid_list",
+  },
```

Comment:
Unless I'm misunderstanding something, why do we want allow the addition of access_limited and auth_bypass_ids to the data for a message in the message queue as these fields will be superfluous to that part of the system?

Reply:
I was just writing they are already sent and will continue to be sent - however I've just realised that because we only send live to the message queue they won't actually need to be sent so can be removed. Thanks I'll sort that out.